cmd/eskip/eskip_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestGetCommandSuccess(t *testing.T) {
+	_, err := getCommand([]string{"some", "print"})
+	if err != nil {
+		t.Error("print is a valid command")
+	}
+}
+
+func TestGetCommandFail(t *testing.T) {
+	_, err := getCommand([]string{"some", "hello"})
+	if err != invalidCommand {
+		t.Error("hello is an invalid command")
+	}
+}
+
+func TestGetCommandEmpty(t *testing.T) {
+	_, err := getCommand([]string{})
```

Comment:
this is used in the tested code with os.Args. os.Args will never be empty, because the command for the os will be always the first arg. Better to test here something like `[]string{"some"}`.